src/cli/README_TCP.md

```diff
@@ -0,0 +1,199 @@
+# OpenThread CLI - TCP Example
+
+The OpenThread TCP APIs may be invoked via the OpenThread CLI.
+
+## Quick Start
+
+### Form Network
+
+Form a network with at least two devices.
+
+### Node 1
+
+On node 1, initialize the TCP CLI module and listen for incoming connections using the example TCP listener.
+
+```bash
+> tcp init
+> tcp listen :: 30000
+```
+
+The `::` specifies the IPv6 Unspecified Address.
+
+### Node 2
+
+On node 2, initialize the TCP CLI module, connect to node 1, and send a simple message.
+
+```bash
+> tcp init
+> tcp connect fe80:0:0:0:a8df:580a:860:ffa4 30000
+> tcp send hello
+```
+
+### Result
+
+After running the `tcp connect` command on node 1, you should see a printout on node 1 similar to below:
+```bash
+TCP: Connection established
+```
+
+After running the `tcp connect` command on node 1, you should see a printout on node 2 similar to below:
```

Comment:
```suggestion
After running the `tcp connect` command on node 2, you should see a printout on node 2 similar to below:
```

Reply:
I think the original text is correct. You only ever run `tcp connect` on node 1 in that example. On node 2, you run `tcp listen` but not `tcp connect`.

Reply:
Ah, yes, I read it too quickly. I think it would be slightly more natural to say something like:
```suggestion
In addition, you should also see a printout on node 2 similar to below:
```
Thoughts?

Reply:
I'm happy with this. I'll update the pull request.